portal/code/cx-portal/src/components/pages/InviteBusinessPartner/index.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  InviteForm,
+  InviteFormData,
+} from 'components/shared/content/InviteForm'
+import { useState } from 'react'
+import UserService from 'services/UserService'
+import './InviteBusinessPartner.scss'
+
+export default function InviteBusinessPartner() {
+  const [processing, setProcessing] = useState<string>('input')
+
+  const doSubmitInvite = (data: InviteFormData) => {
+    setProcessing('busy')
+    fetch(
```

Comment:
why not using httpClient or any axios instance ? this implementation looks bit different than usual way inside the project

Reply:
I copied the logic from the existing portal and only changed it slightly - I will switch to your Axios/HttpClient implementation.